features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/BusinessInformation/BusinessInformation.jsx

```diff
@@ -29,6 +29,7 @@ import { FormattedMessage } from 'react-intl';
 import FormControl from '@material-ui/core/FormControl';
 import API from 'AppData/api.js';
 import { withAPI } from 'AppComponents/Apis/Details/components/ApiContext';
+import AuthManager from '../../../../data/AuthManager';
```

Comment:
Shall we use the `AppData` alias here and all the other places

```suggestion
import AuthManager from 'AppData/AuthManager';
```

Reply:
There are 33 places i guess 😄

Reply:
Fixed with bac9808